apps/windows/package.json

```diff
@@ -15,21 +15,20 @@
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.14.0",
-    "react": "16.11.0",
-    "react-native": "0.62.2",
-    "react-native-windows": "^0.62.1"
+    "react-native": "0.63.4",
+    "react-native-windows": "^0.63.20"
```

Comment:
I dont think we had anything seriously breaking, but be aware that the template app for RNW can change between major versions.

Reply:
Same for Android and iOS for that matter.

Would be work taking a look at react-native-upgrade-helper if you have a moment.

Reply:
We use [React-Native-Test-App](https://github.com/microsoft/react-native-test-app) for the native template code. I have a PR https://github.com/microsoft/fluentui-react-native/pull/552 to move the windows test app as well to use it, but it's blocked on a RNW CLI change that won't go in till RNW 0.64 I believe.

Where could I find the correct template app? I thought react-native-upgrade-helper only did iOS and android?